packages/core/EventManager.ts

```diff
@@ -329,11 +329,10 @@ export default class EventManager {
        *  Not ideal but, if we don't find a destination calendar,
        * fallback to the first connected calendar
        */
-      const [credential] = this.calendarCredentials;
-      if (!credential) {
-        return [];
+      const [credential] = this.calendarCredentials.filter((cred) => cred.type === "calendar");
+      if (credential) {
+        createdEvents.push(await createEvent(credential, event));
```

Comment:
Had to look up if the early returned wouldn't be necessary, but looking at the default on line 309 it seems not. Really smart change, a lot less confusing 🥇

Reply:
I would move the await outside of this .push() sentence.
```suggestion
        const createdEvent = await createEvent(credential, event);
        if (createdEvent) createdEvents.push(createdEvent);

```

Reply:
And probably add a try/catch for that await.